src/spyglass/spikesorting/v1/recording.py

```diff
@@ -0,0 +1,730 @@
+from typing import Tuple, Iterable, Optional, Union, List
+import uuid
+
+import datajoint as dj
+import numpy as np
+import pynwb
+import probeinterface as pi
+import spikeinterface as si
+import spikeinterface.extractors as se
+from neuroconv.tools.spikeinterface.spikeinterfacerecordingdatachunkiterator import (
+    SpikeInterfaceRecordingDataChunkIterator,
+)
+from hdmf.data_utils import GenericDataChunkIterator
+
+from spyglass.common import Session  # noqa: F401
+from spyglass.common.common_ephys import Electrode, Raw  # noqa: F401
+from spyglass.common.common_device import Probe
+from spyglass.common.common_nwbfile import AnalysisNwbfile, Nwbfile
+from spyglass.common.common_interval import (
+    IntervalList,
+    interval_list_intersect,
+)
+from spyglass.common.common_lab import LabTeam
+from spyglass.utils.misc import generate_nwb_uuid
+
+schema = dj.schema("spikesorting_v1_recording")
+
+
+@schema
+class SortGroup(dj.Manual):
+    definition = """
+    # Set of electrodes to spike sort together
+    sort_group_id: str  # identifier for a group of electrodes
+    ---
+    -> Session
```

Comment:
Sorry if I'm missing something, but with Session as a secondary key, doesn't this mean that every sort_group_id needs to be unique, even across Sessions?  

For example, I can't make a `sort_group_id = 0` corresponding to electrode group id 0 for `Winnie20220713_.nwb` and also  a `sort_group_id = 0` corresponding to electrode group id 0 for `Winnie20220714_.nwb` 

Or is the idea to use the naming differently?

Reply:
oops, you're right, I think I should have kept the foreign key from Session as primary key. I will make this change.